src/metrics.rs

```diff
@@ -6,9 +6,12 @@
 //!
 //! To enable metrics collection, call `init_metrics()` before starting the service.
 //!
-//! To record a metric, use the `record!` macro with the metric and the value to record.
-//! To increment a metric by 1, use the `inc!` macro with the metric.
-//! To observe a metric, use the `observe!` macro with the metric and the value to observe.
+//! - To record a **gauge** ( or a **counter**), use the [`record`] macro with a value.
+//!   Don't use gauges though.
+//! - To increment a **counter** by 1, use the [`inc`] macro.
+//! - For **histograms** (or **summaries**, but don't use those either) use the [`observe`]
```

Comment:
Well there are use cases for histograms, and summaries are not properly supported yet.